src/moduleExample/entities/moduleExample.entity.ts

```diff
@@ -2,7 +2,7 @@ import { Entity, Column } from 'typeorm';
 
 import Model from '../../shared/entity.model';
 
-@Entity('posts')
```

Comment:
Please add `pickify_data` or `pickify_posts` schema to this parent class

Reply:
- Added the functionality to create any  entity  in `pickify_posts` schema. 
- However, according to this [issue](https://github.com/typeorm/typeorm/issues/3192): the schema should exist already in the database. So, I added the schema to elephantsql DB.
- When changing elephantSQL to the final DB we will use, we will need to add the required schemas.

Reply:
After testing, yes it could be done with the following CLI command `typeorm query "CREATE SCHEMA pickify_posts"`.
Will be added to README

Reply:
migration script please.

Reply:
Added migration script which runs on each application start. It also creates the schema